.eslintrc.json

```diff
@@ -30,7 +30,8 @@
     "no-shadow": "off",
     "react/display-name": [2, { "ignoreTranspilerName": false }],
     "react/jsx-filename-extension": 0,
-    "react-hooks/exhaustive-deps": 0,
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
```

Comment:
this sort of changes needs an explanation

Reply:
I wanted to use this rule to find out if I was missing anything but it got committed alongside the others.  I will remove them.